Testreglar/214/Nett/nett-2.1.4a.json

```diff
@@ -0,0 +1,196 @@
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Oppgi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			},
+			"kilde": []
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Er det oppgitt av innholdsprodusent eller i innholdet, at testsiden har hurtigtast som består av kun ett tegn?",
+			"ht": "<p>Identifiser hurtigtast som er oppgitt av innholdsprodusent, eller i innholdet.</p>\n<p>Hurtigtast som består av kun ett tegn, omfatter alle utskriftstegn (dvs. alle bokstaver, både store og små), tegnsetting, siffer eller symboler).</p>\n<p><strong>Merk: </strong>Hurtigtast som kombineres med kun <strong>Shift</strong>-tasten er omfattet av kravet, og skal testes. For eksempel, når du bruker Shift-tasten for å få stor bokstav eller tegnsetting.</p>\n<p>Dette skal ikke testes: </p>\n<ul>\n<li>Tilgangstaster (Accesskeys) er ikke omfattet av kravet, fordi de brukes alltid sammen med en eller flere modifiseringstaster.</li>\n<li>iFrame skal ikke testes.</li>\n<li>Kravet gjelder ifølge ordlyden kun i situasjoner der innholdsprodusenten faktisk har tatt i bruk hurtigtaster på nettsiden, som i tillegg består utelukkende av ett enkelt tegn (single-key shortcut). Du skal ikke teste hurtigtaster som er angitt av brukeragenten.</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Har testsiden hurtigtast som består av ett tegn?",
+			"ht": "<p>Slik tester du om det er tatt i bruk hurtigtaster, omfattet av kravet, i tilfeller der vi ikke har opplysninger om det fra innholdsprodusent eller i innholdet: </p>\n<ul>\n<li>Trykk på alle utskriftstegn (Alle bokstaver, både store og små, tegnsetting, siffer eller symbol)\n<ul>\n<li><strong>Merk: </strong>Ikke trykk på\n<ul>\n<li>modifiseringstaster, som Ctrl, Alt, Esc,</li>\n<li>piltaster</li>\n<li>funksjonstaster, F1-F12.</li>\n<li>Space, Enter, Return, Tab eller Delete</li>\n</ul>\n</li>\n</ul>\n</li>\n<li>Se etter endringer på siden som følge av at du har trykket hurtigtasten.</li>\n<li>Hold deretter nede Shift-tasten, trykk på de samme tastene igjen og se etter endringer på siden.</li>\n</ul>",
```

Comment:
Bør hjelpeteksten opplyse om at du noen ganger må stå bestemte steder på nettsiden for at de skal fungere? feks teksteditor eller videospiller